static/js/home_page/custom.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable */
+__webpack_public_path__ = `http://${SETTINGS.host}:8078/`;  // eslint-disable-line no-undef, camelcase
+
+// makes sure the whole site is loaded
+jQuery(window).load(function() {
+	// will first fade out the loading animation

```

Comment:
can you make the tabs consistent in this file? our convention in JS files seems to be 2 spaces for a tab